frontend/src/components/pages/RegistrantExperience/RegistrationSteps/AdditionalInfo/QuestionCards/EDLPSelectionControl.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Radio,
+  RadioGroup,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import React from "react";
+import RequiredAsterisk from "../../../../../common/RequiredAsterisk";
+
+type EDLPSelectionControlProps = {
+  requireEDLP: boolean | null;
+  setRequireEDLP: (setRequireEDLP: boolean) => void;
+  nextClicked: boolean;
+};
+
+const EDLPSelectionControl = ({
+  requireEDLP,
+  setRequireEDLP,
+  nextClicked,
+}: EDLPSelectionControlProps): React.ReactElement => {
+  const invalid = nextClicked && requireEDLP === null;
+
+  const handleMultipleChoiceUpdate = (choice: string) => {
+    setRequireEDLP(choice === "true");
+  };
+
+  return (
+    <FormControl isInvalid={invalid}>
+      <FormLabel textStyle={{ sm: "xSmallBold", lg: "buttonSemiBold" }}>
+        Do your camper(s) require early drop-off or late pick-up?{" "}
+        <Text
+          as="span"
+          color="text.critical.100"
+          fontSize="xs"
+          verticalAlign="super"
+        >
+          <RequiredAsterisk />
+        </Text>
+      </FormLabel>
+      <Text textStyle={{ sm: "xSmallRegular", lg: "buttonRegular" }} mb="3">
+        Note: additional costs will apply. If needed, you may select drop-off
+        and/or pick-up times on the next page{" "}
+      </Text>
+      {invalid && (
+        <FormErrorMessage>Please fill out this question.</FormErrorMessage>
+      )}
+      <RadioGroup
+        value={requireEDLP?.toString()}
+        onChange={(choice) => handleMultipleChoiceUpdate(choice)}
+      >
+        <VStack alignItems="flex-start">
+          <Radio value="true" colorScheme="green">
```

Comment:
doesn't really matter but I feel like setting the radio values to "yes" and "no" would be better than "true" and "false" - this way it aligns with the values we're selecting and there's less confusion when we call setRequireEDLP

Reply:
I think I'll keep it the same since it originally was true/false and I don't want to change it if we don't have to.